botathon/index.php

```diff
@@ -144,14 +144,17 @@
             <div class="range range-md-justify">
                     <div class="cell-md-12 cell-lg-10">
                         <div class="inset-md-right-30 inset-lg-right-0">
-                            <h1><strong>Botathon</strong> - <em>Season 2</em></h1>
-                            <p>Botathon is UNT’s second ever hosted robotics competition, think hackathon+robots!</p>
+                            <h1><strong>Botathon</strong> - <em>Season 2 {2021}</em></h1>
```

Comment:
Can you make it:
`Botathon - Season 2 (20210`
basically, ya need to swap your curly brackets `{ }` for brackets `( )`